### PR TITLE
edits to exceptions pod

### PR DESCRIPTION
**Pull request by antquinonez**

### CODE REVIEW

File: doc/Language/exceptions.pod6 [hunk 1]
```diff
@@ -4,75 +4,78 @@
...
 =head1 Ad hoc exceptions
 
-Ad hoc exceptions work just like in traditional Perl 5, one can simply use
-C<die> with a message as to what went wrong:
+Ad hoc exceptions work just like in Perl 5, where C<die> is called with
+a description of the error.
 
     die "oops, something went wrong";
-    #!> oops, something went wrong in block <unit> at my-script.p6:1
+
```
Comment (Altai-man) [inline doc/Language/exceptions.pod6]: Do we need an empty line in such cases?

File: doc/Language/exceptions.pod6 [hunk 2]
```diff
@@ -174,9 +173,11 @@ to continue from the point of the exception by calling the C<.resume> method.
             when X::AdHoc { .resume }
         }
     }
+
     "OBAI".say;
-    #-> OHAI
-    #-> OBAI
+
+    # -> OHAI
+    # -> OBAI
```
Comment (Altai-man) [inline doc/Language/exceptions.pod6]: We are currently trying to establish a single comment line style, so this changes will be edited soon anyway.

Comment (antquinonez) [inline doc/Language/exceptions.pod6]: @JJ I'll fix that.

Comment (antquinonez) [inline doc/Language/exceptions.pod6]: @JJ fixed.

File: doc/Language/exceptions.pod6 (hunk 1 above)
Comment (antquinonez) [inline doc/Language/exceptions.pod6]: It scanned better for me with a breakline, here and other places.

**Comment by Altai-man** [comment]
Does you built it on localhost? If examples with added new-lines are rendered fine, I'm fine with merging this, though I'm not sure about description text simplifications. It'd be better for someone more confident in English than me to check.

**Pull request merged**